code/utils/calc_t.py

```diff
@@ -0,0 +1,23 @@
+
+import numpy.linalg as npl
+from scipy.stats import mstats
+from scipy.stats import t
+
+def significant(a,b,c):
```

Comment:
Just to be clear it's for one voxel
 a =  entire design matrix X, b = y [:, i], c = beta[:,i] 
And then loop through for each voxel